demo/stories/presets/Presets.stories.tsx

```diff
@@ -1,86 +1,34 @@
 import type {StoryObj} from '@storybook/react';
 
-import {ActionName as Action} from 'src/bundle/config/action-names';
-import {ToolbarName as Toolbar} from 'src/modules/toolbars/constants';
-import {
-    boldItemView,
-    boldItemWysiwyg,
-    colorifyItemMarkup,
-    colorifyItemView,
-    colorifyItemWysiwyg,
-    italicItemMarkup,
-    italicItemView,
-    redoItemMarkup,
-    redoItemView,
-    redoItemWysiwyg,
-    undoItemMarkup,
-    undoItemView,
-    undoItemWysiwyg,
-} from 'src/modules/toolbars/items';
-
 import {Preset as component} from './Preset';
+import {presets, toolbarPresets} from './presets';
+
+const {commonmark, defaultPreset, full, yfm, zero} = presets;
+const {custom} = toolbarPresets;
 
 export const Zero: StoryObj<typeof component> = {
-    args: {preset: 'zero'},
```

Comment:
Something strange: u just replace string literals with constants, but their values ​​are the same string literals. What is point of changes?